src/instana/instrumentation/pep0249.py

```diff
@@ -2,139 +2,161 @@
 # (c) Copyright Instana Inc. 2018
 
 # This is a wrapper for PEP-0249: Python Database API Specification v2.0
-import opentracing.ext.tags as ext
+from __future__ import annotations
 import wrapt
+from typing import TYPE_CHECKING
 
-from ..log import logger
-from ..util.traceutils import get_tracer_tuple, tracing_is_off
-from ..util.sql import sql_sanitizer
+from opentelemetry.semconv.trace import SpanAttributes
+from opentelemetry.trace import SpanKind
+
+from instana.log import logger
+from instana.util.traceutils import get_tracer_tuple, tracing_is_off
+from instana.util.sql import sql_sanitizer
+
+if TYPE_CHECKING:
+    from instana.span.span import InstanaSpan
 
 
 class CursorWrapper(wrapt.ObjectProxy):
-    __slots__ = ('_module_name', '_connect_params', '_cursor_params')
+    __slots__ = ("_module_name", "_connect_params", "_cursor_params")
 
-    def __init__(self, cursor, module_name,
-                 connect_params=None, cursor_params=None):
+    def __init__(
+        self, cursor, module_name, connect_params=None, cursor_params=None
+    ) -> None:
         super(CursorWrapper, self).__init__(wrapped=cursor)
         self._module_name = module_name
         self._connect_params = connect_params
         self._cursor_params = cursor_params
 
-    def _collect_kvs(self, span, sql):
+    def _collect_kvs(self, span, sql) -> InstanaSpan:
         try:
-            span.set_tag(ext.SPAN_KIND, 'exit')
-
-            db_parameter_name = next((p for p in ('db', 'database', 'dbname') if p in self._connect_params[1]), None)
+            span.set_attribute(SpanKind, "exit")
```

Comment:
SpanKind is a class. You can only use predefined span attributes from [here](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-semantic-conventions/src/opentelemetry/semconv/trace/__init__.py). If not present, please type the literal instead.

```suggestion
            span.set_attribute("span.kind", "exit")
```

Reply:
Is there a match table or documentation for that? How can I find the correct attribute?